### PR TITLE
Add text

### PR DESCRIPTION
**Pull request by Cyanne-Jones**
I added some text! Please review it!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The accountabilitrio is the coolest trip in the whole world. Fight me.
```
Comment (hparke1402) [inline README.md:1]: Love this!!!!!!!!!!!!!!!!!

**Review by hparke1402 — APPROVED** [review]
Great Stuff!

File: README.md (hunk 1 above)
Comment (CorCanavan) [inline README.md:1]: Great work!

**Review by CorCanavan — APPROVED** [review]
Looks great to me!

**Pull request merged**